modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, `KnativeEventing`, and the `KnativeKafka` custom resource definitions are installed on your cluster.
```

Comment:
I'd just use "Knative Eventing" here to keep it consistent; we could update to just mention the CRD but we'd need to do it across all the other places it's mentioned too